packages/graphback-codegen-offix/src/OffixPlugin.ts

```diff
@@ -0,0 +1,99 @@
+
+import { GraphbackCoreMetadata, GraphbackPlugin, ModelDefinition, getInputTypeName, getFieldName, GraphbackOperationType } from '@graphback/core'
+import { mergeSchemas } from "@graphql-toolkit/schema-merging"
+import { GraphQLSchema, GraphQLObjectType, GraphQLInt, GraphQLInputObjectType, GraphQLInputObjectTypeConfig, GraphQLField, print, getNullableType, GraphQLNonNull, GraphQLList, printSchema } from 'graphql';
+import { SchemaComposer } from 'graphql-compose';
+
+/**
+ * Configuration for Schema generator CRUD plugin
+ */
+export interface OffixPluginConfig {
+
+    /**
+     * RelativePath for the output files created by generator
+     */
+    outputPath: string
+
+
+    /*
+     * RelativePath for the output files created by generator
+     */
+    // TODO separate annotation for delta queries should be used for more granular support
+    generateDeltaQueries: boolean
+
+    /**
+     * Delta resolvers format
+     */
+    deltaResolverFormat: 'ts' | 'js' | 'graphql',
+
+}
+
+export const SCHEMA_CRUD_PLUGIN_NAME = "SchemaCRUD";
```

Comment:
Should be updated to match the plugin name `OFFIX_PLUGIN_NAME` or similar

Reply:
Good point. We actually ended up not using this at all so this fields do not matter.